docs/setup.rst

```diff
@@ -97,6 +97,17 @@ System requirements
 * Graphical environment (the emulator is based on Pygame)
 * A keyboard (the same keyboard that you're using for the system will work great)
 
+Arch-linux installation
+-----------------------
+.. code-block:: bash
+    sudo pacman -Si python2-pip git python2-pygame
+    sudo pip2 install luma.emulator
+
```

Comment:
I have this feeling that this PR is untested - lack of empty line after the ``.. code-block::`` is IIRC a syntax error =) WIll fix it, no worries.